index.js

```diff
@@ -124,6 +124,7 @@ module.exports = {
 
       if (this.uikitOptions.useIcons) {
         this.app.import(path.join('vendor', 'uikit-icons.js'));
+        this.app.import(path.join('vendor', 'shims', 'uikit.js'));
```

Comment:
This makes no sense to me.. Why only import the shim if we use icons?

Reply:
Sorry about that, I pasted it on the wrong line when I made a feature branch for these changes. I updated the RR.